packages/basti/src/target/elasticache-cluster/elasticache-cluster-init-target.ts

```diff
@@ -10,24 +10,24 @@ import { InitTargetBase } from '../init-target.js';
 import type { CacheCluster } from '@aws-sdk/client-elasticache';
 
 export class ElasticacheClusterInitTarget extends InitTargetBase {
```

Comment:
Please, rename this class to `ElasticacheRedisClusterInitTarget` just like you did with the connect target. Rename files as well.